src/Vision/Annotate.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Vision;
+
+use Google\Cloud\Vision\Connection\ConnectionInterface;
+
+class Annotate
+{
+    private $connection;
+    private $images;
+
+    public function __construct(ConnectionInterface $connection, array $images)
+    {
+        $this->connection = $connection;
+        $this->images = $images;
+    }
+
+    public function annotate(array $options = [])
```

Comment:
I borrowed some of the API from `gcloud-node`, which uses `annotate`. I see the point though, so I'll see if I can come up with anything better, and am open to suggestions.